modules/gdscript/language_server/gdscript_text_document.cpp

```diff
@@ -281,8 +281,6 @@ Dictionary GDScriptTextDocument::resolve(const Dictionary &p_params) {
 }
 
 Array GDScriptTextDocument::foldingRange(const Dictionary &p_params) {
-	Dictionary params = p_params["textDocument"];
-	String path = params["uri"];
```

Comment:
CC @Geequlim. Is it expected that this method returns an empty Array?

Reply:
@akien-mga Looks good to me.
FoldingRange is implemented in script languge clients so we just return an empty Array here.

Reply:
Thanks!